### ISSUE TITLE
Prefetch Custom textures support

### ISSUE THREAD

**Issue by pepodmc** [reicast/reicast-emulator#1497]
Hello, Can you add an option to prefetch Custom textures?

So the emulator can load all textures in the beginning of the emulation (like in dolphin emulator). Because some textures doesnt load instantly and looks bad when you can see the change from the original texture to the new texture ingame.

Thanks!

**Comment by Emulatorer** [comment]
Duplicate  #895? 🤔

**Comment by pepodmc** [comment]
> Duplicate #895?

no, that one was talking about custom texture support, that was added recently (january 2019 and the issue was from 2016)
And im talking about prefetch custom textures, a different thing.

**Comment by Emulatorer** [comment]
Okey
